README.md

```diff
@@ -108,47 +72,74 @@ With this technology we want to **achieve**:
    npm run dev
    ```
 
+<br/>
+
+## 💻 Technology Used
+
+Qiskit.org is a static website generated using [Nuxt](https://nuxt.com/).
+
+We create and run tests using [Vitest](https://vitest.dev/), avoid syntax errors using [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/), and automate code integration and deployment using [GitHub Actions](https://github.com/features/actions).
+
+With these technologies we aim to achieve the following:
+
+- Using a component-based framework (like Vue) that allows us to reuse the UI code in different parts of the application.
+- Fast initial page load.
+- Index content on Search Engines.
+- Test JS unit functions.
+- Avoid syntax errors.
+- Continuous integration pipeline.
+
+<br/>
+
 ## 🏭 Content Generation
 
-qiskit.org integrates with the tools used by the IBM Quantum Community Team and generates some content based on 3rd party APIs such as Airtable. Part of this content is prefetched during building time. While developing, it is disabled by default. If you want enable content generation, you must set the environment variable `GENERATE_CONTENT`. For instance:
+qiskit.org integrates with the tools used by the IBM Quantum Community Team to generate content based on 3rd party APIs such as Airtable. Part of this content is prefetched during building time. While developing, it is disabled by default. If you want enable content generation, you must set the environment variable `GENERATE_CONTENT`. For instance:
 
 ```shell
 GENERATE_CONTENT=1 npm run dev
 ```
 
-Notice that, for communicating with the team tools, API keys may be required. It is the case of dealing with Airtable for the generation of the event index. If you think you should have access to these tables, talk to the Event Squad in the Community Team, get your developer API key and set the `AIRTABLE_API_KEY` environment variable to this value:
+Notice that, for communicating with the team tools, API keys may be required. It is the case of dealing with Airtable for the generation of the event index. If you think you should have access to these tables, talk to the Events Squad in the Community Team, get your developer API key and set the `AIRTABLE_API_KEY` environment variable to this value:
 
 ```shell
 GENERATE_CONTENT=1 AIRTABLE_API_KEY=<your airtable api key> npm run dev
 ```
+<br/>
 
-## 🧐 Directory structure
+## 📁 Directory structure
```

Comment:
IMO, this section should be removed since it doesn't offer much value.

We are following Nuxt's directory structure, but that's to be expected since we already mentioned that we're using Nuxt.

Reply:
Removed in https://github.com/Qiskit/qiskit.org/pull/3172/commits/e663ede4b578cbd2c4034f7a29d7e5b5588449d1